spec/requests/admin/configs_spec.rb

```diff
@@ -79,6 +79,33 @@
           expect(SiteConfig.authentication_providers).to eq([provider])
         end
 
+        it "allows all providers to be disabled" do
+          # We have to send _something_ for an update, so we'll simulate the borwser omitting the auth
+          # providers array when no options are selected
+          post "/admin/config", params: { site_config: {ga_tracking_id: "123" },
+                                          confirmation: confirmation_message }
+          expect(SiteConfig.authentication_providers).to be_empty
+        end
+
+        context "when authentication providers are present" do
+          before do
+            SiteConfig.authentication_providers = Authentication::Providers.available.map(&:to_s)
```

Comment:
This needs to be updated, it has been removed in master: https://github.com/forem/forem/pull/10811

I believe nothing after line 89 should show up in the diff.